addons/account/models/reconciliation_widget.py

```diff
@@ -265,6 +265,7 @@ def get_data_for_manual_reconciliation(self, res_type, res_ids=None, account_typ
         assert account_type in ('payable', 'receivable', None)
         is_partner = res_type == 'partner'
         res_alias = is_partner and 'p' or 'a'
+        aml_ids = self._context.get('active_ids') and tuple(self._context.get('active_ids'))
```

Comment:
shouldn't we also check context.get('active_model') to be sure that the active_ids are coming from account_move_line and not from another source?